### PR TITLE
Update eval_iter_loop.ipynb

### PR DESCRIPTION
**Pull request by Hakimovich99**

### CODE REVIEW

File: src/pipeline_eval.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ def create_pipeline(  # noqa: PLR0913
     pipeline_dir: str = "./data-dir",
     embed_model_provider: str = "huggingface",
     embed_model: str = "all-MiniLM-L6-v2",
+    api_keys: dict = {},
```
Comment (RobbeSneyders) [inline src/pipeline_eval.py:10]:
```suggestion
    embed_api_key: dict = {},
```

Can we rename this to make clear it's for the embedding component?

File: src/utils.py [hunk 2]
```diff
@@ -28,7 +28,36 @@ def create_directory_if_not_exists(path):
     return str(p_base_path)
 
 
-# Read latest chosen component
+# Store pipeline results
+def store_results(  # noqa: PLR0913
```
Comment (RobbeSneyders) [inline src/utils.py:32]: I guess ideally this would be a component in the pipeline?
Fine for now to keep it like this.

**Review by RobbeSneyders — COMMENTED** [review]
Thanks @Hakimovich99 

Can you rename the api key variable? Then it's good to go for me.

File: src/utils.py (hunk 2 above)
Comment (Hakimovich99) [reply src/utils.py:32]: Didn't think about it but yes could be!

File: src/pipeline_eval.py (hunk 1 above)
Comment (Hakimovich99) [reply src/pipeline_eval.py:10]: sure!

**Review by RobbeSneyders — APPROVED** [review]

**Pull request merged**